src/shared/dict.js

```diff
@@ -0,0 +1,7 @@
+export default function makeDict() {
```

Comment:
this is no longer needed, new v8 makes this the default for `Object.create(null)`